### PR TITLE
feat(github): add agent atomics for AI agents

### PR DESCRIPTION
**Pull request by ibrahim-abuznaid**
Adds a set of `audience: 'ai'` agent actions to the **GitHub** piece — atomic, single-purpose operations designed for AI agents, complementing the existing actions used in the visual flow builder.

## Changes
- **75 new `audience: 'ai'` actions** for the agent surface (issues, pull requests, comments, repositories, branches, files/contents, releases, labels, collaborators, etc.)
- **10 existing actions re-tagged `audience: 'human'`** where a new agent atomic supersedes them — the agent uses the atomic variants; the human action picker is otherwise unchanged
- Version `0.8.3` → `0.8.4`

## Verification
- `tsc -p packages/pieces/community/github/tsconfig.lib.json` — clean
- `eslint` — 0 errors
- Tier-2 (live): exercised against a real GitHub account during authoring (the large majority verified live; a couple require org/repo permissions not available on the test account)

Branched off fresh `upstream/main`.

### CODE REVIEW

**Comment by ibrahim-abuznaid** [comment]
Thanks — all four addressed in the latest commit.

- **Unescaped file paths (3×P1):** `get-file-content`, `create-or-update-file`, and `delete-file` now encode each path segment (`path.split('/').map(encodeURIComponent).join('/')`), so file names containing URL-reserved characters resolve to the correct Contents API path.
- **Milestone clear unrepresentable (P1):** `Property.Number` can't carry `null`, so I added a `Clear Milestone` checkbox that sends `milestone: null` when set (overriding the number); description updated to point at it.

Typecheck + lint clean. Live re-test to follow.

**Comment by ibrahim-abuznaid** [comment]
Status: the path-encoding and Clear Milestone fixes are typecheck/lint-clean and code-reviewed. The specific edge cases (file names with URL-reserved characters; clearing an already-assigned milestone) were not exercised against a live connection in this pass.

**Comment by ibrahim-abuznaid** [comment]
@greptileai review

**Comment by ibrahim-abuznaid** [comment]
@greptileai

**Comment by ibrahim-abuznaid** [comment]
Update: both fixes are now live-verified against a real GitHub connection. Created and read a file at a path containing a reserved character (greptile-enc/a#b.md) and it round-trips to the correct path; and set, then cleared, an issue milestone via the new Clear Milestone option (the milestone becomes null). The edge cases I flagged earlier as not-yet-exercised are confirmed working.

File: packages/pieces/community/github/src/lib/actions/ai/update-issue-ai.ts [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    if (propsValue.title !== undefined) body['title'] = propsValue.title;
+    if (propsValue.body !== undefined) body['body'] = propsValue.body;
+    if (propsValue.state !== undefined) body['state'] = propsValue.state;
+    if (propsValue.state_reason !== undefined)
+      body['state_reason'] = propsValue.state_reason;
+    if (propsValue.clear_milestone) {
+      body['milestone'] = null;
+    } else if (propsValue.milestone !== undefined) {
+      body['milestone'] = propsValue.milestone;
+    }
```
[the remark this answers is not in the document]
Comment (ibrahim-abuznaid) [reply packages/pieces/community/github/src/lib/actions/ai/update-issue-ai.ts:108]: Not a bug in the Activepieces execution path. Property.Checkbox inputs are strictly validated by the engine before run() executes: props-processor.ts validates CHECKBOX with `typeof value === 'boolean'` (there is no checkbox coercion processor in processors/index.ts), and piece-executor.ts throws on any validation error before calling run(). A string "false" would be rejected at validation and never reach `if (propsValue.clear_milestone)`, so that check only ever sees a real boolean or undefined. No change required.

**Review by sanket-a11y — APPROVED** [review]
